run.sh

```diff
@@ -31,3 +31,5 @@ echo "** DONE **"
 grep "^DELETE" < generated/dbupdate.log | awk '{s+=$2} END {print "Deleted: " s}'
 grep "^INSERT" < generated/dbupdate.log | awk '{s+=$3} END {print "Added: " s}'
 tail -1 generated/allocator-update.log
+
+rm -rf generated
```

Comment:
I am afraid this does not work for me. I am often using the `generated` files to search for deals when troubleshooting miner complaints.

I propose to gate this by a new ENV var. The default behaviour can be deleting the files; I am fine with that.

```suggestion
[ -z "$KEEP_GENERATED" ] && rm -rf generated
```

(Plus, an update in README?)

Reply:
if you use these folders, then it's simpler not to add this and just let me add the `rm -rf` myself